client/apps/settings/test/client/curations/venice_admin.coffee

```diff
@@ -81,8 +79,7 @@ describe 'VeniceSection', ->
   it 'Clicking save button saves the curation and removes warnings from save button', ->
     r.simulate.click r.findTag(@component, 'button')[0]
     @component.state.isSaving.should.eql true
-    @component.state.curation.save.callCount.should.eql 1
-    setTimeout( =>
-      @component.state.isSaving.should.eql false
-      @component.state.isChanged.should.eql false
-    , 750)
```

Comment:
Generally we should avoid using `setTimeout` in tests, especially without a `done` callback. This looks to be the main issue.